scripts-vagrant_provision/prerequisites.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+sudo apt-get update
+# For NVM
+sudo apt-get install -y curl
+# For pyenv
+sudo apt-get install -y git build-essential libssl-dev zlib1g-dev libbz2-dev libreadline-dev libsqlite3-dev wget curl llvm libncurses5-dev libncursesw5-dev xz-utils tk-dev libffi-dev liblzma-dev python-openssl
+
+NODE_VERSION=16
```

Comment:
```suggestion
NODE_VERSION=18
```
16 is EOL already since this week

Reply:
yes but until we update to 20.04 I think it is better to keep it this way